instagram/src/App.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import './App.css';
+import dummyData from './dummy-data';
+import SearchBar from './components/SearchBar/SearchBar.js';
+import PostListContainer from './components/PostListContainer/PostListContainer.js';
+import PostsPage from './components/PostContainer/PostsPage.js';
+
+
+class App extends Component {
+
+  constructor(){
+    super();
+    this.state = {
+      postData: []
+
+    }
+  }
+
+  componentDidMount(){
+    setTimeout(() => {
+      this.setState({postData: dummyData});
+
+    }, 500);
```

Comment:
Nice job tweaking this to update on half-second intervals. Makes the search seem super fast for the user 🔥